stdlib/array.mli

```diff
@@ -230,9 +259,10 @@ val sort : ('a -> 'a -> int) -> 'a array -> unit
    compare as equal, a positive integer if the first is greater,
    and a negative integer if the first is smaller (see below for a
    complete specification).  For example, {!Stdlib.compare} is
-   a suitable comparison function.  After calling [Array.sort], the
+   a suitable comparison function, provided there are no floating-point
+   NaN values in the data.  After calling [sort], the
```

Comment:
It is the label side that is inexact here: `compare` is suitable even in presence of multiple NaN: see also https://github.com/ocaml/ocaml/pull/1936/files#r228982988

Reply:
Good catch. Will fix.